release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -557,6 +557,13 @@ For more information, see xref:../networking/k8s_nmstate/k8s-nmstate-updating-no
 
 {product-title} {product-version} now includes CoreDNS version 1.11.3.
 
+[id="ocp-4-17-expose-mtu"]
+==== Exposing MTU for vfio-pci SR-IOV devices
+
+With this release, MTU on virtual function using the vfio-pci driver is available in the network-status pod annotation, and inside the container. 
+
+For more information, see xref:../networking/hardware_networks/add-pod.html#nw-sriov-expose-mtu_configuring-sr-iov[Exposing MTU for vfio-pci SR-IOV devices to pod].
```

Comment:
This link leads to a page about "Adding a pod to an SR-IOV additional network", is this the location you were intending to point to?